tests/nnfw_api/src/CircleGen.cc

```diff
@@ -43,6 +43,13 @@ uint32_t CircleGen::addTensor(const TensorParams &params)
   return ind;
 }
 
+uint32_t CircleGen::addTensor(const TensorParams &params, const SparsityParams &sp)
+{
+  int ind = curSubgCtx().tensors.size();
```

Comment:
(optional) 
```suggestion
  uint32_t ind = curSubgCtx().tensors.size();
```

Reply:
@mhs4670go Thank you. I've updated. (including `addTensor` with 1 parameter version ­­— which the code is originated.)